packages/ods/src/components/combobox/tests/rendering/ods-combobox.e2e.ts

```diff
@@ -228,4 +306,4 @@ describe('ods-combobox rendering', () => {
       expect(hasScroll).toBe(false);
     });
   });
-});
+});
```

Comment:
check your IDE to avoid removing last line automatically (or we'll have a diff each time)